.travis.yml

```diff
@@ -63,6 +63,7 @@ matrix:
       env: TEST_SELENIUM="true" SCALA_VER="2.10" SPARK_VER="1.6.1" HADOOP_VER="2.3" PROFILE="-Pspark-1.6 -Phadoop-2.3 -Ppyspark -Pexamples" BUILD_FLAG="package -DskipTests -DskipRat" TEST_FLAG="verify -DskipRat" TEST_PROJECTS="-pl zeppelin-interpreter,zeppelin-zengine,zeppelin-server,zeppelin-display,spark-dependencies,spark -Dtest=org.apache.zeppelin.AbstractFunctionalSuite -DfailIfNoTests=false"
 
 before_install:
+  echo "MAVEN_OPTS=-Xmx2048m" >> ~/.mavenrc
```

Comment:
(Ref - https://docs.travis-ci.com/user/customizing-the-build/#Installing-Packages-Using-apt)